src/cmd/exporter/main.go

```diff
@@ -21,17 +22,27 @@ func main() {
 	viper.SetEnvPrefix("harbor")
 	viper.AutomaticEnv()
 	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+	connMaxLifetime, err := time.ParseDuration(viper.GetString("database.conn_max_lifetime"))
+	if err != nil {
+		connMaxLifetime = 0
```

Comment:
Print error log and should override the value to 5m if user input is invalid.

Reply:
@chlins Done.